torchgeo/datasets/idtrees.py

```diff
@@ -595,27 +594,12 @@ def plot_las(self, index: int, colormap: Optional[str] = None) -> Any:
         points: "np.typing.NDArray[np.int_]" = np.stack(
             [las.x, las.y, las.z], axis=0
         ).transpose((1, 0))
+        point_cloud = pyvista.PolyData(points)  # type: ignore[attr-defined]
```

Comment:
Same question here, would be happy to add type hints for this if needed

Reply:
Even if I import `from pyvista import PolyData` and change the line to `point_cloud: PolyData = PolyData(points)` mypy still complains `Module "pyvista" has no attribute "PolyData"  [attr-defined]`